### DISCUSSION TITLE
My selected school has incorrect or incomplete information e.g. domains or campus location

### DISCUSSION THREAD

**Discussion by CyferBoy** [community/community#113243] [GitHub Education]
### Hi everyone,

I am currently enrolled in IGNOU's online program of post-graduation. and trying to get github student access but it says "You appear not to be near any campus location for the school you have selected." but in an online mode that is not necessory please if anyone have any idea please let me know.

i selected ignou delhi 2 where CEO is my study center.

IGNOU Is 100% Distance learning program every course available on ignou is distance mode.

![image](https://github.com/community/community/assets/70091085/49473489-309b-4ed0-809d-b65a19a229b4)

**Comment by AidenJamesYt** [comment]
if you have a vpn on disable it

**Reply by CyferBoy** [reply]
I don't have any vpn enabled

**Comment by samus-aran** [comment]
Hi @CyferBoy 👋 

We can see you've opened a support ticket. The community is unable to assist with nor escalate your ticket but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

Couple of tips on submitting:

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request **students submit a live photo of their evidence rather than an upload**. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

**Reply by CyferBoy** [reply]
I tried about 8 times since June 23 everytime same error. Location error is persistent. And when I tried to take photo from back camera it doesn't switch only showing the front camera. Option is available but when I select nothing happens, this makes more difficult to take good photo.

**Reply by MadhavKwatra** [reply]
@CyferBoy any updates here? I too have the same issues. I have enrolled in IGNOU Postgrad program. On phone , it still the same issue , cannot use back camera.

**Reply by AshishBuildit** [reply]
> @CyferBoy any updates here? I too have the same issues. I have enrolled in IGNOU Postgrad program. On phone , it still the same issue , cannot use back camera.

did you got any update brother?

**Reply by CyferBoy** [reply]
Unfortunately, no it still not working in my phone
